### DISCUSSION TITLE
Missing features

### DISCUSSION THREAD

**Discussion by erolbozkurt** [sumatrapdfreader/sumatrapdf#3955] [Ideas]
Dude, if you add "highlight the selected text" feature, so that I can take notes while reading, I can pay for it ;o)

**Comment by GitHubRulesOK** [comment]
Highlight was there in experimental versions you could add highlight in text ebook and pdf etc that still work in official versions up to 3.2 
but during 3.2-3.3 development the "cross format" highlight was considered problematic so those 4 switched to PDF and included as start of 
> "3.3 (2021-07-06)
added support for adding / removing / editing annotations in PDF files. Read [the tutorial](https://www.sumatrapdfreader.org/docs/Editing-annotations)"
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/703ab107-e016-40b0-9894-7426211211ee)
